### PR TITLE
Secure vault doc refactoring

### PR DESCRIPTION
**Pull request by omindu**

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -702,7 +702,7 @@ nav:
       - 'Securing Passwords in Configuration Files':
         - 'Encrypting Passwords with Cipher Tool': administer/encrypting-passwords-with-cipher-tool.md
         - 'Resolving Encrypted Passwords': administer/resolving-encrypted-passwords.md
-        - 'Carbon Secure Vault Implementation': administer/carbon-secure-vault-implementation.md
+        - 'Customizing Secure Vault': administer/customizing-secure-vault.md
```
Comment (gomathyk) [inline en/mkdocs.yml]: Please check if there are links from other files to this and replace all available instances with the new name.

File: en/docs/administer/encrypting-passwords-with-cipher-tool.md [hunk 2]
```diff
@@ -73,43 +71,15 @@ To change any password that we have encrypted already, follow the below steps:
...
-If you start the Micro Integrator as a background job, you will not be able to provide password values on the command line. Therefore, you must start the server in "daemon" mode as explained below.
-
-1. Create a new file in the <IS_HOME> directory. The file should be named according to your OS as explained below.
-
-    * For Linux: The file name should be `password-tmp`.
-    * For Windows: The file name should be `password-tmp.txt`.
+## What's next?
 
-2. Add the primary keystore password (which is "wso2carbon" by default) to the new file and save. By default, the password provider assumes that both private key and keystore passwords are the same. If not, the private key password must be entered in the second line of the file.
+For information on resolving the encrypted passwords see [Resolving Encrypted Passwords](../../administer/resolving-encrypted-passwords).
```
Comment (YvonneW) [inline en/docs/administer/encrypting-passwords-with-cipher-tool.md]: Let's add this to an infobox as shown below:

!!! info
    For information on xxxx, see xxx .

File: en/docs/administer/encrypting-passwords-with-cipher-tool.md [hunk 3]
```diff
@@ -73,43 +71,15 @@ To change any password that we have encrypted already, follow the below steps:
...
 
-3. Now, start the server as a background process by running the following command.
-   ```
-   ./server.sh start
-   ```
-4. Start the server by running the product start-up script from the MI_HOME/bin directory by executing the following command:
-   ```
-   daemon. sh server.sh -start
-   ```
+For information on customizing secure vault implementaion see [Customizing Secure Vault](../../administer/customizing-secure-vault).
```
Comment (YvonneW) [inline en/docs/administer/encrypting-passwords-with-cipher-tool.md]: Let's add this to an infobox as shown below:

!!! info
    For information on xxxx, see xxx .

**Comment by omindu** [comment]
Comments are addressed

**Pull request closed without merging**